_install-and-configure/install-dashboards/tls.md

```diff
@@ -13,12 +13,14 @@ By default, for ease of testing and getting started, OpenSearch Dashboards runs
 
```

Comment:
There are some other settings you can find here: https://github.com/opensearch-project/OpenSearch-Dashboards/blob/28c1fbb84550d33a036e231661867988af5ebe54/src/cli/serve/serve.js#L102

Which may be worth adding to this doc if not present elsewhere.

Reply:
added info about more parameters